usr/share/rear/output/USB/Linux-i386/850_make_USB_bootable.sh

```diff
@@ -30,7 +30,13 @@ LogPrint "Making $RAW_USB_DEVICE bootable with syslinux/extlinux"
 # so we search for " $BUILD_DIR/outputfs " in /proc/mounts to get the filesystem (third field) where the booting related files are:
 usb_filesystem=$( grep " $BUILD_DIR/outputfs " /proc/mounts | cut -d' ' -f3 | tail -1 )
 case "$usb_filesystem" in
-    (ext?)
+    # Installing extlinux should also be working for a mounted vfat boot partition
+    # see https://github.com/rear/rear/issues/2884
+    # because https://wiki.archlinux.org/title/syslinux reads (excerpts):
+    #   For FAT, ext2/3/4, or btrfs boot partition use extlinux, where the device has been mounted:
+    #     # extlinux --install ...
+    #   Alternatively, for a FAT boot partition use syslinux, where the device is unmounted
+    (ext?|vfat)
         if [[ "$FEATURE_SYSLINUX_EXTLINUX_INSTALL" ]] ; then
             extlinux -i "$BUILD_DIR/outputfs/$SYSLINUX_PREFIX" || Error "'extlinux -i $BUILD_DIR/outputfs/$SYSLINUX_PREFIX' failed"
```

Comment:
the synopsis of extlinux91) manual page says "extlinux - install the SYSLINUX bootloader on a ext2/ext3 filesystem". So, I am a bit surprised that it supports also FAT, but i have tried `extlinux -i /mnt` on a RHEL 7 system, where `/mnt` is a FAT filesystem, and it passed (I have not tried booting from it though). So it apparently supports FAT as well and the support is not new (@jsmeix  you could try even older SLES versions).

Reply:
By the way, i am curious what is the `extlinux` command without flags below supposed to do. I get:
```
# extlinux  /mnt
Usage: extlinux [options] directory
  --install    -i  Install over the current bootsector
  --update     -U  Update a previous installation
  --zipdrive   -z  Force zipdrive geometry (-H 64 -S 32)
  --sectors=#  -S  Force the number of sectors per track
  --heads=#    -H  Force number of heads
  --stupid     -s  Slow, safe and stupid mode
  --raid-mode  -r  Fall back to the next device on boot failure
  --once=...   -o  Execute a command once upon boot
  --clear-once -O  Clear the boot-once command
  --reset-adv      Reset auxilliary data
  --menu-save= -M  Set the label to select as default on the next boot
````
So it does not like to be invoked without any flags, at least not on RHEL 7.

Reply:
I had read "man extlinux" (syslinux-4.04 in openSUSE Leap 15.4)
and had noticed therein the
```
extlinux - install the SYSLINUX bootloader on a ext2/ext3 filesystem
```
but because of what there is on
https://wiki.archlinux.org/title/syslinux
I concluded the man page is just outdated.

And when the extlinux installation fails because
an older syslinux version does not support extlinux on VFAT,
then ReaR will error out with an error message
so the user is still informed similar as in
https://github.com/rear/rear/issues/2884

Reply:
@pcahyna 
regarding calling "the extlinux command without flags":

I think the FEATURE_SYSLINUX_EXTLINUX_INSTALL code
is from old times for old syslinux / extlinux
but I won't like to touch that code and clean it up
because for me that code is too complicated because
I am not a sufficient SYSLINUX expert to rewrite that code
so I prefer to not touch that code as far as possible.

Reply:
@jsmeix @hpannenb thank you for the explanations and references.